src/nvim/os/input.c

```diff
@@ -61,10 +61,11 @@ void input_start_stdin(int fd)
 void input_stop_stdin(void)
 {
   if (!read_stream) {
-    return;
+    input_start_stdin(0);
```

Comment:
Under what situation will `read_stream == NULL` without running through this function and calling `rstream_set_blocking`? As far as I can tell, the only place it could be set `NULL` would be here and on initialization. If it was set here, `stdin` should be back in blocking mode. If it's `NULL` from `input_start_stdin()` having never been called, `stdin` should have never been set to non-blocking. Does libuv change the flags, even if we don't use `stdin`?

I read @oni-link's comment, but I don't understand why this happens.